pkg/controller/kogitoruntime/kogitoruntime_controller.go

```diff
@@ -105,6 +105,7 @@ func (r *ReconcileKogitoRuntime) Reconcile(request reconcile.Request) (reconcile
 		OnDeploymentCreate: onDeploymentCreate,
 		OnObjectsCreate:    onObjectsCreate,
 		OnGetComparators:   onGetComparators,
+		CanBeBuilt:         true,
```

Comment:
is this correct? I mean, shouldn't it come from parameter or something else?

Reply:
It's coming from the parameter. The Service Deployer API is highly customizable and for this use case, we need to know if the service is a "custom" one or not. Do you have any other suggestion for this name? It's not a parameter that a user would inform.

Reply:
Removed this parameter and renamed the builder.